lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -975,6 +1001,68 @@ def configure_network(self, vm_obj):
                 self.configspec.deviceChange.append(nic)
                 self.change_detected = True
 
+    def configure_vapp_properties(self, vm_obj):
```

Comment:
Can you rename some variables ; for example  vappProperties could be vapp_properties_current, propsToChange could be vapp_properties_to_change…
Avoid variable like x, k, v, tmp… unless you are in one line expression like `dict((x.id, x) for i in …)`.

Reply:
agreed

Reply:
Fixed